src/test/java/com/ably/kafka/connect/ChannelSinkTaskIT.java

```diff
@@ -56,55 +55,236 @@ public void setup() throws Exception {
+        connectCluster.start();
+        connectCluster.assertions().assertAtLeastNumWorkersAreUp(NUM_WORKERS, "Initial group of workers did not start in time.");
 
         // connect Ably client
         ablyClient = AblyHelpers.realtimeClient(appSpec);
     }
 
     @AfterEach
     public void close() {
-        connect.stop();
+        connectCluster.stop();
         ablyClient.close();
         AblyHelpers.deleteTestApp(appSpec);
     }
 
+    //channel name tests
     @Test
-    public void testMessagePublish() throws Exception {
-        // create a test Kafka topic
-        connect.kafka().createTopic("kafka-connect-ably-test");
+    public void testMessagePublish_channelExistsWithStaticChannelName() throws Exception {
+        final String channelName = "test-channel";
+        // topic1
+        final String topic = TOPICS.split(",")[0];
+        connectCluster.kafka().createTopic(topic);
 
-        // configure the Ably connector to publish messages from the test Kafka
-        // topic to an Ably channel
-        Map<String, String> settings = new HashMap<>();
-        settings.put(CONNECTOR_CLASS_CONFIG, SINK_CONNECTOR_CLASS_NAME);
-        settings.put(TASKS_MAX_CONFIG, String.valueOf(NUM_TASKS));
-        settings.put(TOPICS_CONFIG, "kafka-connect-ably-test");
-        settings.put(KEY_CONVERTER_CLASS_CONFIG, ByteArrayConverter.class.getName());
-        settings.put(VALUE_CONVERTER_CLASS_CONFIG, ByteArrayConverter.class.getName());
-        settings.put(ChannelSinkConnectorConfig.CHANNEL_CONFIG, "kafka-connect-ably-test");
-        settings.put(ChannelSinkConnectorConfig.CLIENT_KEY, appSpec.key());
-        settings.put(ChannelSinkConnectorConfig.CLIENT_ID, "kafka-connect-ably-test");
-        settings.put(ChannelSinkConnectorConfig.CLIENT_ENVIRONMENT, AblyHelpers.TEST_ENVIRONMENT);
-        connect.configureConnector(CONNECTOR_NAME, settings);
-        connect.assertions().assertConnectorAndAtLeastNumTasksAreRunning(CONNECTOR_NAME, NUM_TASKS, "Connector tasks did not start in time.");
+        Map<String, String> settings = createSettings(channelName, null, null, null);
+        connectCluster.configureConnector(CONNECTOR_NAME, settings);
+        connectCluster.assertions().assertConnectorAndAtLeastNumTasksAreRunning(CONNECTOR_NAME, NUM_TASKS, "Connector tasks did not start in time.");
+
+        // subscribe to the Ably channel
+        Channel channel = ablyClient.channels.get(channelName);
+        AblyHelpers.MessageWaiter messageWaiter = new AblyHelpers.MessageWaiter(channel);
+
+        // produce a message on the Kafka topic
+        connectCluster.kafka().produce(topic, "foo", "bar");
+
+        // wait 5s for the message to arrive on the Ably channel
+        messageWaiter.waitFor(1, TIMEOUT);
+        final List<Message> receivedMessages = messageWaiter.receivedMessages;
+        assertEquals(receivedMessages.size(), 1, "Unexpected message count");
```

Comment:
This is repeated in the other tests so maybe we could extract it to a new method? :thinking:  Something like:
```java
assertReceivedExactAmountOfMessages(1);
```

Reply:
This is also a good idea 👍 a3fe71d06dd5f3c30903c2f87239772f864bcf3e